stories/topics/proc-aws-managing-backups-playbook.adoc

```diff
@@ -0,0 +1,93 @@
+
+== Delete backups
+.Procedure
+. Populate the `command_generator_data` directory with the configuration file template.
++
+[source,bash]
+----
+docker run --rm -v $(pwd)/command_generator_data/:/data $IMAGE command_generator_vars aws_backups_delete --output-data-file /data/backups.yml
+----
++
+Produces the following output:
++
+[literal, options="nowrap" subs="+quotes,attributes"]
+----
+===============================================
+Playbook: aws_backups_delete
+Description: This playbook delete a specified backup.
+-----------------------------------------------
+This playbook delete a specified backup
+
+-----------------------------------------------
+Command generator template: 
+
+docker run --rm -v <local_data_file_directory>:/data $IMAGE command_generator aws_backups_delete --data-file /data/backups.yml
+----
+. After running the command, a `/tmp/backups.yml` template file is created. 
+This template file resembles the following: 
++
+[literal, options="nowrap" subs="+quotes,attributes"]
+----
+aws_backups_delete:
+  cloud_credentials_path:
+  extra_vars:
+    aws_backup_names:
+    aws_region:
+    aws_s3_bucket:
+    delete:
+----
+
+With backup_names like ["backup1","backup2"]
+
+. To run the deletion of the backups, run the command generator to generate the deletion backup command.
++
+[literal, options="nowrap" subs="+quotes,attributes"]
+----
+docker run --rm -v $(pwd)/command_generator_data:/data $IMAGE command_generator aws_backups_delete --data-file /data/backups.yml
+----
++
+Resulting in the following ouput:
++
+[literal, options="nowrap" subs="+quotes,attributes"]
+----
+Command to run playbook: 
+
+docker run --rm --env PLATFORM=AWS -v ~/.aws/credentials:/home/runner/.aws/credentials:ro \
+--env ANSIBLE_CONFIG=../aws-ansible.cfg  $IMAGE redhat.ansible_on_clouds.aws_backups_delete \
+-e 'aws_region=<region> aws_s3_bucket=<bucket> aws_backup_names=["mybackup"] delete=True'
+===============================================
+----
+. Run the supplied backup command to trigger the backup deletion.
++
+. When the playbook has finished running, the output resembles the following:
++
+[literal, options="nowrap" subs="+quotes,attributes"]
+----
+TASK [redhat.ansible_on_clouds.standalone_aws_backup_delete : [delete_backup] Dry-run message] ***
+skipping: [localhost]
+
+PLAY RECAP *********************************************************************
+localhost                  : ok=21   changed=2    unreachable=0    failed=0    skipped=2    rescued=0    ignored=0    
+----
+
+
+= Troubleshootig
+
+== Failed to delete a backup
+
+If the deletion of backup failed, take the following steps.
+
+1. Go to the bucket containing the backup and locate the directory having the backup name.
+2. Open that directory
+3. Open the file names "restore-vars.json" and note last part of the "Recovery_Point_Arn" and the "Backup_Vault_Name"
```

Comment:
`restore-vars.json ` is one file. Does it contain both peices of information.

Reply:
yes